### ISSUE TITLE
[first] doesn't recognize "import = require" imports in TypeScript

### ISSUE THREAD

**Issue by abacabadabacaba** [benmosher/eslint-plugin-import#1957]
**Configuration**:
```javascript
module.exports = {
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint', 'import'],
  rules: {
    'import/first': 'error'
  }
}
```

**Code**

```ts
import foo from 'foo'
import bar = require('bar')
import baz from 'baz'
```

**Expected result**

No errors.

**Actual result**

```
3:1  error  Import in body of module; reorder to top  import/first
```
